packages/pointer_interceptor/pointer_interceptor_ios/ios/Classes/PointerInterceptorView.swift

```diff
@@ -0,0 +1,26 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+import Flutter
+import UIKit
+
+public class PointerInterceptorView: NSObject, FlutterPlatformView {
+
+  let interceptorView: UIView
+
+  init(frame:CGRect, debug:Bool) {
+
+    interceptorView = UIView(frame:frame)
+    if debug {
+      interceptorView.backgroundColor = UIColor(red: 1, green: 0, blue: 0, alpha: 0.5)
+    } else {
+      interceptorView.backgroundColor = UIColor.clear
+    }
```

Comment:
Optional: consider using a ternary here since the operation the same and only the value is different:
```swift
interceptorView.backgroundColor = debug ? UIColor(red: 1, green: 0, blue: 0, alpha: 0.5) : UIColor.clear
```